[2023-05-17T16:46:14.070+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T16:46:14.081+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T16:46:14.082+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T16:46:14.100+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-17T16:46:14.107+0000] {standard_task_runner.py:57} INFO - Started process 1047 to run task
[2023-05-17T16:46:14.110+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpp227x2ry']
[2023-05-17T16:46:14.111+0000] {standard_task_runner.py:85} INFO - Job 22: Subtask clean_fondue_data
[2023-05-17T16:46:14.167+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 8cf8fb9e28f7
[2023-05-17T16:46:14.258+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-17T16:46:14.330+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T16:46:14.365+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T16:46:14.376+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230517T164614, end_date=20230517T164614
[2023-05-17T16:46:14.403+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T16:46:14.434+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
<<<<<<< HEAD
[2023-05-17T17:55:01.182+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T17:55:01.201+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T17:55:01.202+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T17:55:01.225+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-17T17:55:01.236+0000] {standard_task_runner.py:57} INFO - Started process 704 to run task
[2023-05-17T17:55:01.240+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpqpgaj310']
[2023-05-17T17:55:01.243+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask clean_fondue_data
[2023-05-17T17:55:01.316+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host a37c7bc3731c
[2023-05-17T17:55:01.458+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-17T17:55:01.576+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T17:55:01.633+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T17:55:01.648+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230517T175501, end_date=20230517T175501
[2023-05-17T17:55:01.694+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T17:55:01.730+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:18:11.277+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T18:18:11.298+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T18:18:11.299+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:18:11.555+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-17T18:18:11.571+0000] {standard_task_runner.py:57} INFO - Started process 847 to run task
[2023-05-17T18:18:11.586+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpd46ghp0s']
[2023-05-17T18:18:11.594+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask clean_fondue_data
[2023-05-17T18:18:12.166+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 3390ff13fbf6
[2023-05-17T18:18:12.665+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-17T18:18:12.751+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:18:13.161+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:18:13.176+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230517T181811, end_date=20230517T181813
[2023-05-17T18:18:13.879+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:18:13.918+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:25:46.791+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T18:25:46.807+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T18:25:46.808+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:25:46.835+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-17T18:25:46.846+0000] {standard_task_runner.py:57} INFO - Started process 811 to run task
[2023-05-17T18:25:46.850+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmp6or4opil']
[2023-05-17T18:25:46.852+0000] {standard_task_runner.py:85} INFO - Job 23: Subtask clean_fondue_data
[2023-05-17T18:25:46.923+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 70fcd41af239
[2023-05-17T18:25:47.062+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-17T18:25:47.209+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:25:47.251+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:25:47.265+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230517T182546, end_date=20230517T182547
[2023-05-17T18:25:47.303+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:25:47.353+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-17T18:37:29.902+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T18:37:29.928+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-17T18:37:29.930+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-17T18:37:29.970+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-17T18:37:29.983+0000] {standard_task_runner.py:57} INFO - Started process 1463 to run task
[2023-05-17T18:37:29.987+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmp228h0yku']
[2023-05-17T18:37:29.991+0000] {standard_task_runner.py:85} INFO - Job 34: Subtask clean_fondue_data
[2023-05-17T18:37:30.084+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 1545bed90dab
[2023-05-17T18:37:30.270+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-17T18:37:30.414+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-17T18:37:30.489+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-17T18:37:30.509+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230517T183729, end_date=20230517T183730
[2023-05-17T18:37:30.610+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-17T18:37:30.693+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-18T06:41:12.845+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T06:41:12.866+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T06:41:12.867+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:41:12.903+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-18T06:41:12.914+0000] {standard_task_runner.py:57} INFO - Started process 1211 to run task
[2023-05-18T06:41:12.918+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpj90quzwb']
[2023-05-18T06:41:12.922+0000] {standard_task_runner.py:85} INFO - Job 32: Subtask clean_fondue_data
[2023-05-18T06:41:13.086+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 714a0c7ad307
[2023-05-18T06:41:13.306+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-18T06:41:13.441+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:41:13.497+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:41:13.509+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230518T064112, end_date=20230518T064113
[2023-05-18T06:41:13.533+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:41:13.712+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-05-18T06:53:22.737+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T06:53:22.760+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T06:53:22.761+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T06:53:22.807+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-18T06:53:22.819+0000] {standard_task_runner.py:57} INFO - Started process 1226 to run task
[2023-05-18T06:53:22.823+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpyaqp4aza']
[2023-05-18T06:53:22.827+0000] {standard_task_runner.py:85} INFO - Job 30: Subtask clean_fondue_data
[2023-05-18T06:53:22.923+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 26547302378e
[2023-05-18T06:53:23.190+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-18T06:53:23.313+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T06:53:23.382+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T06:53:23.400+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230518T065322, end_date=20230518T065323
[2023-05-18T06:53:23.439+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T06:53:23.488+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
=======
[2023-05-18T04:22:15.760+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T04:22:15.784+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T04:22:15.785+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T04:22:15.870+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-18T04:22:15.882+0000] {standard_task_runner.py:57} INFO - Started process 474 to run task
[2023-05-18T04:22:15.886+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpt3dk0n09']
[2023-05-18T04:22:15.888+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask clean_fondue_data
[2023-05-18T04:22:16.009+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host b36de7cfc389
[2023-05-18T04:22:16.192+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-18T04:22:16.344+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T04:22:16.408+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T04:22:16.442+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230518T042215, end_date=20230518T042216
[2023-05-18T04:22:16.746+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T04:22:16.797+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
>>>>>>> 8e4a41b368593e9ce283ae02afd6c27252f67370
[2023-05-18T07:21:08.542+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:08.544+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:08.623+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:08.626+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:08.699+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T07:21:08.730+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T07:21:08.731+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T07:21:08.769+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-18T07:21:08.786+0000] {standard_task_runner.py:57} INFO - Started process 1941 to run task
[2023-05-18T07:21:08.791+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpm6pkys1_']
[2023-05-18T07:21:08.795+0000] {standard_task_runner.py:85} INFO - Job 37: Subtask clean_fondue_data
[2023-05-18T07:21:08.924+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:21:08.926+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:21:08.932+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 0a50b96f7e57
[2023-05-18T07:21:09.179+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-18T07:21:09.385+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T07:21:09.589+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T07:21:09.617+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230518T072108, end_date=20230518T072109
[2023-05-18T07:21:09.690+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T07:21:09.770+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-05-18T07:38:27.854+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:27.856+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:27.898+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:27.899+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:27.946+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T07:38:27.964+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [queued]>
[2023-05-18T07:38:27.965+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-18T07:38:27.994+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): clean_fondue_data> on 2023-05-09 00:00:00+00:00
[2023-05-18T07:38:28.007+0000] {standard_task_runner.py:57} INFO - Started process 844 to run task
[2023-05-18T07:38:28.013+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'fondue_dag', 'clean_fondue_data', 'scheduled__2023-05-09T00:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/fondue_dag.py', '--cfg-path', '/tmp/tmpbcel9rku']
[2023-05-18T07:38:28.017+0000] {standard_task_runner.py:85} INFO - Job 26: Subtask clean_fondue_data
[2023-05-18T07:38:28.156+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509
[2023-05-18T07:38:28.157+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=fondue_dag/run_id=scheduled__2023-05-09T00:00:00+00:00/task_id=clean_fondue_data'
[2023-05-18T07:38:28.162+0000] {task_command.py:410} INFO - Running <TaskInstance: fondue_dag.clean_fondue_data scheduled__2023-05-09T00:00:00+00:00 [running]> on host 4fe746fadf66
[2023-05-18T07:38:28.323+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='Bhuribhat@gmail.com' AIRFLOW_CTX_DAG_OWNER='pras' AIRFLOW_CTX_DAG_ID='fondue_dag' AIRFLOW_CTX_TASK_ID='clean_fondue_data' AIRFLOW_CTX_EXECUTION_DATE='2023-05-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-09T00:00:00+00:00'
[2023-05-18T07:38:28.460+0000] {logging_mixin.py:149} INFO - error lol
[2023-05-18T07:38:28.586+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-05-18T07:38:28.610+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=fondue_dag, task_id=clean_fondue_data, execution_date=20230509T000000, start_date=20230518T073827, end_date=20230518T073828
[2023-05-18T07:38:28.672+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-18T07:38:28.731+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
